partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/DiadicComparisonOperator.kt

```diff
@@ -39,7 +39,7 @@ internal abstract class DiadicComparisonOperator(name: String) : DiadicOperator(
 
     abstract fun getNumberComparison(lhs: Number, rhs: Number): Boolean
 
-    private fun Datum.getNumber(): Number {
+    internal fun Datum.getNumber(): Number {
```

Comment:
Maybe we ought to move this extension function?

Reply:
Yea, moved to a different Datum util file as an extension function [`3cfeb1d` (#1755)](https://github.com/partiql/partiql-lang-kotlin/pull/1755/commits/3cfeb1df6b0ec2dde306c075bcf9f12798e5eb0f)